src/assets/crubrand/generic/_tools.scss

```diff
@@ -17,12 +19,12 @@ $rem-base: 16px !default;
  */
 // scss-lint:disable NameFormat 
 @function parseInt($n) { /* 2 */
-    @return $n / ($n * 0 + 1);
+    @return math.div($n, ($n * 0 + 1));
 }
 // scss-lint:enable NameFormat
 
 @function convert-to-rem($value, $base-value: $rem-base) {
-    $value: strip-units($value) / strip-units($base-value) * 1rem;
+    $value: math.div(strip-units($value), strip-units($base-value) * 1rem);
```

Comment:
I don't think order of operations affects this but I probably would have had the ` * 1rem` after the `math.div` function call instead of as the second parameter.